spacy_llm/tasks/entity_linking.py

```diff
@@ -0,0 +1,310 @@
+import csv
+import re
+import warnings
+from pathlib import Path
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Type, Union
+
+import jinja2
+import spacy
+from pydantic import BaseModel
+from spacy.language import Language
+from spacy.pipeline import EntityLinker
+from spacy.scorer import Scorer
+from spacy.tokens import Doc, Span
+from spacy.training import Example
+
+from ..registry import registry
+from ..ty import CandidateSelector, ExamplesConfigType
+from .templates import read_template
+from .util import SerializableTask
+
+_DEFAULT_EL_TEMPLATE_V1 = read_template("entity_linking")
+
+
+class EntityLinkingExample(BaseModel):
+    text: str
+    mentions_str: str
+    mentions: List[str]
+    entity_descriptions: List[List[str]]
+    entity_ids: List[List[str]]
+    solutions: List[str]
+    reasons: Optional[List[str]]
+
+
+@registry.llm_misc("spacy.CandidateSelectorPipeline.v1")
```

Comment:
Let's try and think of less verbose names for this one ;-)

Reply:
Renamed to `spacy.CandidateSelector.v1` in https://github.com/explosion/spacy-llm/pull/241/commits/0e7a28d74e30ddb6950933c0227cd92df9b4af94. Feel free to close if you think that's alright.